src/resolvers/FridgeItemResolver.ts

```diff
@@ -108,4 +109,51 @@ export class FridgeItemResolver {
 			};
 		}
 	}
+
+	/**
+	 * Fetches all fridge items from a certain fridge
+	 *
+	 * @param fridgeId the id of the fridge
+	 * @return all fridge items of the fridge with the specified fridgeId
+	 */
+	@Query(() => FridgeItemsResponse)
+	async getFridgeFridgeItems(
+		@Arg('fridgeId') fridgeId: number
+	): Promise<FridgeItemsResponse> {
+		let fridgeItems;
+		try {
+			const getFridgeItems = await client.query(
+				`
+				SELECT * FROM "fridgeItems"
+				WHERE "fridgeItems"."fridgeId" = $1
+				`,
+				[fridgeId]
+			);
+			if (getFridgeItems == undefined) {
+				return {
+					errors: [
+						{
+							field: 'table: fridgeItems',
+							message: 'database returned undefined',
+						},
+					],
+				};
+			}
+
+			fridgeItems = getFridgeItems.rows;
+		} catch (err) {
+			return {
+				errors: [
```

Comment:
We repeat this block of code in plenty of places. We should make a shared function for this.